docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.md

```diff
@@ -230,24 +246,24 @@ class PriceProductToProductFacadeBridge extends SprykerPriceProductToProductFaca
 }
 ```
 
-Related error in the Evaluator output:
-
-
-```bash
-------------------------------------------------------------------------------------
-Please avoid extension of the PrivateApi Spryker\Zed\PriceProduct\Dependency\Facade\PriceProductToProductFacadeBridge in Pyz\Zed\PriceProduct\Dependency\Facade\PriceProductToProductFacadeBridge
-------------------------------------------------------------------------------------
-```
-
-
+#### Example of resolving the error
 To resolve the error, remove the bridge.
 
+---
 
 ## PrivateApi:MethodIsOverridden
 
+It is not allowed to override the protected core methods from the core level on the project level. Protected methods can be changed at any time.
```

Comment:
What are we trying to explain by the second sentence?

Reply:
done